runtime/include/komp.h

```diff
@@ -172,9 +172,21 @@ class __cplus_omp_init_lock_t
  */
 typedef void ident_t;
 
+#ifndef __WORDSIZE
+#ifdef _MSC_VER
+#ifdef _WIN64
+#define __WORDSIZE 64
+#else
+#define __WORDSIZE 32
+#endif
+#else
+#error "Unknown word size!"
```

Comment:
Does this still compile correctly for MinGW?
Afaict, `__WORDSIZE` isn't defined for that platform either. IIUC, it is only defined on platforms with glibc.

Reply:
Sadly, I don't have any MinGW machine available at present. __WORDSIZE isn't only defined on platforms with glibc (e.g. XCode defines it in stdint.h, which is covered by the next pull request). If you can verify that the following code fails to compile on MinGW, please let us know:
```
#include <stdint.h>

#ifndef __WORDSIZE
#error "__WORDSIZE not defined"
#endif
```

Reply:
With MINGW64 (from MSYS2):
```
$ cat wordsize.c
#include <stdint.h>

#ifndef __WORDSIZE
#error "__WORDSIZE not defined"
#endif

Markus@Markus-PC MINGW64 ~
$ gcc wordsize.c
wordsize.c:4:2: error: #error "__WORDSIZE not defined"
    4 | #error "__WORDSIZE not defined"
      |  ^~~~~
```

Reply:
See also #1271.